Packs/DomainTools_Iris/IndicatorFields/indicatorfield-domainage.json

```diff
@@ -0,0 +1,31 @@
+{
+    "id": "indicator_domainage",
+    "version": -1,
+    "modified": "2023-10-16T17:13:01.885961592Z",
+    "name": "Domain Age",
```

Comment:
Prefix needed

Reply:
@Ni-Knight apologies I'm not sure what you mean for this one. Do you have a link to docs, or a good example I can follow here?

Reply:
I updated the id, name, cliName fields and filename with a prefix of `domaintoolsiris`. Let me know if you had something else in mind here.